x-pack/plugins/security_solution/server/endpoint/services/agent/clients/crowdstrike/types.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface RawCrowdstrikeInfo {
+  crowdstrike: {
+    host: {
+      id: string;
+      last_seen: string;
+      hostname: string;
+    };
+    reduced_functionality_mode: 'no'; // or else?
+    status: string;
+  };
+  cid: string;
+  device: {
+    id: string;
+  };
```

Comment:
Can you revisit this type definition and sync with the Crowdstrike integration. A few properties here do not look correct to me.  Several property are not defined in the correct structure.